sample/db/samples/reimbursements.rb

```diff
@@ -7,6 +7,11 @@
 stock_location = inventory_unit.find_stock_item.stock_location
 return_reason = Spree::ReturnReason.active.take!
 preferred_reimbursement_type = Spree::ReimbursementType.where(name: 'Original').take!
+admin_user = if defined?(Spree::Auth)
+  Spree.user_class.admin.take!
+else
+  Spree.user_class.find_or_create_by!(email: 'admin@example.com')
+end
```

Comment:
This is a clever way to set an user, but if we need an actual admin user, shouldn't we create it with more information than just the email?

Reply:
This will be executed when `Auth::Devise` is not present, so it will create a LegacyUser which does not need any more information to be valid.